specification/gp-connect-access-record-structured-fhir.yaml

```diff
@@ -82,14 +82,24 @@ info:
 
     - you must have a clinical safety officer (CSO) who is responsible for [DCB0129](https://digital.nhs.uk/data-and-information/information-standards/information-standards-and-data-collections-including-extractions/publications-and-notifications/standards-and-collections/dcb0129-clinical-risk-management-its-application-in-the-manufacture-of-health-it-systems) and, if necessary, [DCB0160](https://digital.nhs.uk/data-and-information/information-standards/information-standards-and-data-collections-including-extractions/publications-and-notifications/standards-and-collections/dcb0160-clinical-risk-management-its-application-in-the-deployment-and-use-of-health-it-systems) (for more on clinical risk management, visit [Clinical risk management standards](https://digital.nhs.uk/services/clinical-safety/clinical-risk-management-standards))
     If you are confident that you can meet the prerequisites, please express an interest with the GP Connect team (see 'Onboarding' below).
+    
+    Become an API consumer
+    If you're planning on consuming data using GP Connect APIs then you're a consumer system.
+
+    Become an API provider
+    If you're planning on providing data using GP Connect APIs then you're a provider system.
     ## Related APIs
     The following APIs are also related to this API:
 
     - GP Connect Access Document - FHIR API - use this API to retrieve unstructured documents from a patient's GP practice record
     - Personal Demographics Service - FHIR API  - use this API to search for patients, retrieve patients by NHS number and update patients
     - Personal Demographics Service - HL7 V3 API  - use this if you want to use functions that are not yet available on the FHIR API
-    - Spine Directory Service - FHIR API  - access accredited system information and messaging endpoint details
+    - Spine Directory Service (SDS) - FHIR API  - access accredited system information and messaging endpoint details
```

Comment:
I know the APIs above aren't linked but it would be good to add that to all four of them (potentially another PR for that).